dialogue-core/src/test/java/com/palantir/dialogue/core/BalancedNodeSelectionStrategyChannelTest.java

```diff
@@ -127,25 +128,118 @@ void constant_4xxs_do_eventually_move_the_needle_but_we_go_back_to_fair_distribu
 
         for (int i = 0; i < 11; i++) {
             channel.maybeExecute(endpoint, request);
-            assertThat(channel.getScores())
+            assertThat(channel.getScoresForTesting().map(c -> c.getScore()))
                     .describedAs("%s %s: Scores not affected yet %s", i, Duration.ofNanos(clock.read()), channel)
                     .containsExactly(0, 0);
             incrementClockBy(Duration.ofMillis(50));
         }
         channel.maybeExecute(endpoint, request);
-        assertThat(channel.getScores())
+        assertThat(channel.getScoresForTesting().map(c -> c.getScore()))
                 .describedAs("%s: Constant 4xxs did move the needle %s", Duration.ofNanos(clock.read()), channel)
                 .containsExactly(1, 0);
 
         incrementClockBy(Duration.ofSeconds(5));
 
-        assertThat(channel.getScores())
+        assertThat(channel.getScoresForTesting().map(c -> c.getScore()))
```

Comment:
peanut gallery: I want an error-prone rule that can refactor this automatically
```suggestion
        assertThat(channel.getScoresForTesting().map(BalancedNodeSelectionStrategyChannel::getScore))
```

Reply:
That would be pretty cool! Think we just need to improve https://github.com/palantir/gradle-baseline/blob/15661f333a76997d0a5b79a52af8767e01a86ca1/baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/LambdaMethodReference.java#L45 to handle more than suppliers

Reply:
there ya go: https://github.com/palantir/gradle-baseline/pull/1359